sdcm/cluster.py

```diff
@@ -3241,6 +3250,16 @@ def get_datacenter_name_per_region(self, db_nodes=None):
 
         return datacenter_name_per_region
 
+    def get_rack_names(self, db_nodes=None):
+        rack_names_mapping = {}
+        for rack, nodes in self.nodes_by_racks(nodes=db_nodes).items():
+            if status := nodes[0].get_nodes_status():
```

Comment:
Can't we get_nodes_status just once before for loop?

Reply:
I guess we can, I've copied the logic from the `per_region_dc` function